apps/promisetracker/contrib/dokku/Dockerfile

```diff
@@ -1 +1 @@
-FROM codeforafrica/promise-tracker-ui:0.2.0-alpha.6
+FROM codeforafrica/promise-tracker-ui:0.2.0-alpha.5-next
```

Comment:
Why are we going backwards here?

Reply:
I think `package.json` should be the main source of the version so that's what was picked up.

Reply:
Then lets be careful; moving backwards will cause a clash in the future when we get to .6 again since the image with that version number will already be in docker hub.

PS: I think it will best to change versioning of PromiseTracker in this repo vs the one in the standalone repo just in case we need to make a quick fix in the standalone repo... May be we should go for `0.3.0-alpha.1` for this one since we know we **_won't_** continue feature development in the standalone repo.